controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -186,6 +192,25 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           };
         }
 
+        const count = await fedGraphRepo.count();
```

Comment:
Please provide a few integration tests for all the new features.

Reply:
Moving this into a separate issue.